CRM/Event/WorkflowMessage/ParticipantTrait.php

```diff
@@ -50,6 +50,17 @@ trait CRM_Event_WorkflowMessage_ParticipantTrait {
    */
   public $isShowParticipantCount;
 
+  /**
+   * What is the participant count, if 'specifically configured'.
+   *
+   * See getter notes.
+   *
+   * @var bool
+   *
+   * @scope tplParams as participantCount
+   */
+  public $participantCount;
```

Comment:
`r-code`: This hurt my brain a little bit -- it declares a concrete/public property, but I don't see anything which reads or writes the property (`$this->participantCount`).

* This looks like it may be a (somewhat quirky) implementation of a virtual/computed property? i.e. You can declare `@scope tplParams as participantCount`; and then implement `getParticipantCount()` to do computation; and the *effect* is to export the computed value. But it's misleading because there's a (public) property that can be read+written in other ways.
* If that's the idea,then... we  should probably support `@scope` directly on the method `getParticipantCount()` so that there's no misleading property. (*Obviously, a bigger patch.*)
* If this is a pattern that you've been using, then... ehm... ok... I guess it's not a hard block on this change... but maybe at least....
    1. Mark the property as `protected`... (For purposes of `MagicGetterSetter`s and `importAll()/exportAll()`,  [public and protected should both be equally valid](https://github.com/civicrm/civicrm-core/blob/0de5ee422e427e6957469938c91664af03a5c9dd/Civi/Api4/Utils/ReflectionUtils.php#L171-L172).)
    2. Expand on the comment "See getter notes"... Maybe: "This property is never actually stored. It is a virtual property computed by the getter."

Reply:
@totten this is consistent with the other properties used in the workflow message classes